serdect/tests/cbor.rs

```diff
@@ -9,12 +9,16 @@ use serde::Serialize;
 use serdect::{array, slice};
 
 /// Example input to be serialized.
-const EXAMPLE_BYTES: [u8; 16] = hex!("000102030405060708090A0B0C0D0E0F");
+/// Last byte is `0xFF` to test that no packing is performed for values under 128.
+const EXAMPLE_BYTES: [u8; 16] = hex!("000102030405060708090A0B0C0D0EFF");
 
 /// CBOR serialization of [`EXAMPLE_BYTES`] as a slice.
-const CBOR_SLICE: [u8; 17] = hex!("90000102030405060708090A0B0C0D0E0F");
+/// (first three bits, `0b010` denote a byte string, and the last five, `0b10000` denote the length)
+const CBOR_SLICE: [u8; 17] = hex!("50000102030405060708090A0B0C0D0EFF");
 
 /// CBOR serialization of [`EXAMPLE_BYTES`] as an array.
+/// (first three bits, `0b100` denote an array, and the last five, `0b10000` denote the length)
+/// Note the 0x18 marker before 0xFF, denoting that the integers are dynamically sized.
```

Comment:
I think this whole comment has to be adjusted now.

Reply:
Fixed